tracer/test/test-applications/debugger/Samples.Probes/probes_definition.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file probably wasn't meant to be here and should be deleted. Sorry for that - it's an artifact of running the debugger's tests, which we probably should have put under `.gitignore`. We'll address this in a separate PR.

Reply:
thanks for the info it  s good to know!